app/controllers/follow_requests_controller.rb

```diff
@@ -52,8 +53,9 @@ def update
   def destroy
     @follow_request.destroy
     respond_to do |format|
-      format.html { redirect_back fallback_location: root_url, notice: "Follow request was successfully destroyed." }
+      format.html { redirect_back fallback_location: root_url}
```

Comment:
Thanks for the comment! 
I realize that since our notice is placed at the top, most users don't see it. I'm considering switching to an alert or modal feature in the future. :)